### DISCUSSION TITLE
내 bluesky bridge 계정이 작동되지 않는다

### DISCUSSION THREAD

**Discussion by CodeNameLay** [snarfed/bridgy-fed#1655] [Q&A]
My bluesky bridge hasn't been working for a while now. I've changed my account name, blocked and unblocked it, but it still doesn't seem to be working. Any idea why? 
I want my Misskey followers to see my bluesky posts.. (I used a translator)

**Comment by snarfed** [comment]
Hey, sorry for the trouble! Have you looked at https://fed.brid.gy/docs#troubleshooting ?

**Comment by MimzyFang** [comment]
sounds like https://github.com/snarfed/bridgy-fed/issues/1570 to me
